README.md

```diff
@@ -42,13 +43,16 @@ Done! Now you should be able to access the project at **0.0.0.0:8080** or **loca
 
 ## Documentation
 
-- Please follow the rules mention in the [contributor's document](https://github.com/siv2r/kidney-exchange/blob/master/CONTRIBUTING.md) while contributing to this project.
-- [How to create a pull request?](https://github.com/siv2r/kidney-exchange/blob/master/CreatePR.md)
+- Please follow the rules mention in the [contributor's document](https://github.com/siv2r/kidney-exchange/blob/master/docs/CONTRIBUTING.md) while contributing to this project.
+- [How to create a pull request?](https://github.com/siv2r/kidney-exchange/blob/master/docs/CreatePR.md)
```

Comment:
since the file location is changed this link is not working. I think `(docs/CreatePR.md)` will fix it. 
more info here: https://stackoverflow.com/questions/11868512/linking-using-relative-path-on-github-readme-md